READMEs.md

```diff
@@ -1,105 +1,70 @@
-README standards
-================
-
-There are a number of things that people immediately want/need to know when they look at a repo:
-* What does it do?
-* How do I build it?
-* How do I run it?
-
-This information should be available in the README, following the standard GFM (github flavoured markdown) template below.
-Its worth noting that this is the basic information that needs to be available, but there will quite likely be more useful information that is required per repo so make sure you add that too.
-
-
-General GFM style pointers
---------------------------
-
-* Use code blocks where necessary
-* If a section goes off the page, then link to it from the top of the page (see #mongo-requirement in the template below - by default section headings automatically become anchors with the lowercased wording of the title and spaces changed to dashes - so "Mongo Requirement" becomes #mongo-requirement)
-* Use sub README's where required i.e. if the repo has a "scripts" directory then it should have an accompanying README to describe what the scripts do, how to use them, etc.
-* Always link to sub README's from the main/top level README
-
-To help with support in service repos there should be subordinate READMEs where appropriate linked to the main README as follows
+# ReadMe Standards
```

Comment:
Would benefit from Markdownlint - it would say it needs a new line here. I can get it plugged in if you want?

Reply:
yes please but you might need to explain it to me